ui/de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/DiskResourceMetadataViewImpl.java

```diff
@@ -122,171 +80,108 @@ public void render(Context context, String value, SafeHtmlBuilder sb) {
     private String baseId;
 
     public DiskResourceMetadataViewImpl(boolean isEditable) {
-
         alc = new AccordionLayoutContainer();
         centerPanel = new VerticalLayoutContainer();
-        timestampFormat = DateTimeFormat.getFormat(DateTimeFormat.PredefinedFormat.DATE_TIME_SHORT);
+
 
         writable = isEditable;
         valid = true;
-        templateStore = new ListStore<>(new TemplateInfoModelKeyProvider());
 
         initWidget(uiBinder.createAndBindUi(this));
         con.setCenterWidget(centerPanel);
         initGrid();
         addMetadataButton.setEnabled(writable);
         deleteMetadataButton.disable();
-        templateCombo.setEnabled(writable);
     }
 
     @Override
-    public DiskResourceMetadataTemplate getMetadataTemplate() {
-        if (selectedTemplate == null) {
-            return null;
-        }
-
-        DiskResourceMetadataTemplate metadataTemplate = autoBeanFactory.templateAvus().as();
-        metadataTemplate.setId(selectedTemplate.getId());
-
-        ArrayList<DiskResourceMetadata> avus = Lists.newArrayList();
-
-        for (String attr : templateAttrFieldMap.keySet()) {
-            DiskResourceMetadata avu = templateAttrAvuMap.get(attr);
-            if (avu == null) {
-                avu = newMetadata(attr, "", ""); //$NON-NLS-1$ //$NON-NLS-2$
-                templateAttrAvuMap.put(attr, avu);
-            }
-
-            Field<?> field = templateAttrFieldMap.get(attr);
-            if (field.getValue() != null) {
-                String value = field.getValue().toString();
-                if ((field instanceof DateField) && !Strings.isNullOrEmpty(value)) {
-                    value = timestampFormat.format(((DateField) field).getValue());
-                } else if(field instanceof ComboBox<?>) {
-                    @SuppressWarnings("unchecked")
-                    ComboBox<TemplateAttributeSelectionItem> temp = (ComboBox<TemplateAttributeSelectionItem>)field;
-                    value = temp.getValue().getValue();
-                }
-
-                avu.setValue(value);
-            }
-
-            avus.add(avu);
-        }
-        metadataTemplate.setAvus(avus);
-
-        return metadataTemplate;
+    public void mask() {
+    	con.mask();
     }
+    
+    @Override
+    public void unmask() {
+    	con.unmask();
+    }
+    
 
     @Override
     public List<DiskResourceMetadata> getAvus() {
-        return listStore.getAll();
+        return additionalMdListStore.getAll();
     }
 
     @Override
     public boolean isValid() {
-        if (selectedTemplate != null && templateForm != null) {
+/*        if (selectedTemplate != null && templateForm != null) {
             List<IsField<?>> fields = FormPanelHelper.getFields(templateForm);
             for (IsField<?> f : fields) {
                 if (!f.isValid(false)) {
                     valid = false;
                 }
             }
 
-        }
-        return valid;
+        }*/
+        return false;
```

Comment:
Since this method now always returns `false`, I was wondering if it could possibly go away. It appears that this method is still called from `MetadataDialog`, which is still used by `PermanentIdRequestViewImpl`. It would seem that the confirmation message box would always be displayed when attempting to close a permanent ID request dialog now unless I'm missing something. If this is a problem, would it be useful to restore this method or possibly to modify `PermanentIdRequestViewImpl` to use a different metadata dialog box instead?